### ISSUE TITLE
TinyCME and CKeditor fiedls won't be translated

### ISSUE THREAD

**Issue by thomassausen** [digitalpulsebe/craft-multi-translator#26]
Craft CMS 5.4.2
Multi-Translator 2.7.0

The client used the TinyMCE field, which was translatable, but the plugin didn't translate into other languages. Standard text fields were fine. We changed to the latest CKEDitor and do still have the problems, that everything will be translated beside the CKEditor fields.

The field is set to "translateable for every language".

**Comment by JodebaDigitalPulse** [comment]
Hi @thomassausen ,

TinyMCE is indeed not in the list of [supported fields](https://github.com/digitalpulsebe/craft-multi-translator?tab=readme-ov-file#supported-field-types) right now. I'll be looking in to it to add it to the list.

CKEditor should be fine, I will test it with your versions to be sure.

**Comment by JodebaDigitalPulse** [comment]
Hi @thomassausen, 

Translating CKEditor fields should still be working.

If you create this file: config/multi-translator.php with contents:

return [
    "debug" => true,
];

You will have some logged entries in storage/logs for each translation. Maybe have a look there, or send the logs?
If you see the field's content in the serialized dump, then the text is processed.
What API service are you using? There might be an edge in the content itself as well.

**Comment by JodebaDigitalPulse** [comment]
Hi, you can send the logs to support@digitalpulse.be, thank you!

**Comment by JodebaDigitalPulse** [comment]
Hi @thomassausen ,

There is no html field in your Entrytype directly.
![image](https://github.com/user-attachments/assets/6d08ccdc-38cd-4919-b89b-c507e9c434ed)

So I'm guessing it is inside the matrix field? This for example?
![image](https://github.com/user-attachments/assets/13fff008-7509-440f-8599-7f6b2eb0c930)

So, the field might be translatable, and as you can see a translated value is processed. But one thing I can't see from the logs is the **propagation method** of the Matrix field itself. I suggest you select 'Only save blocks to the site they were created in'

**Comment by thomassausen** [comment]
Sorry, I thought you would reply via mail :D 

Okay, yes CKEditor field in Matrix is set to translate for each language. And the propagation method of the Matrix field itself was set to "Save entries to all sites the owner element is saved in" and I changed it to your suggestion.

**Comment by thomassausen** [comment]
Okay, it worked. I went into the old article, where the Italian article mistakenly had the German translation. I deleted the Italian translation so that the field was empty. Of course, the German article still contained the German text.

Then I selected the article in the entry overview and selected the “Translate to Italian” button at the bottom. And then the field was also translated.

The field must probably be empty and not yet contain any text if you activate the automatic translation. This would also explain why it worked directly for new articles.
